Source/View Controller/MapViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  MapViewController.swift
+//  KnightHacks
+//
+//  Created by Patrick Stoebenau on 11/10/19.
+//  Copyright © 2019 KnightHacks. All rights reserved.
+//
+
+import UIKit
+
+internal class MapViewController: UIViewController, UIScrollViewDelegate {
+    @IBOutlet weak var mapImage: UIImageView!
+    @IBOutlet weak var scrollView: UIScrollView!
+
+    @IBAction func closeButton(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
```

Comment:
This should be at the bottom

Reply:
I moved this to the bottom as you suggested.